tests/createServer.test.js

```diff
@@ -1,206 +1,73 @@
+'use strict';
 /* eslint-disable max-len */
 
-const { Server, get } = require('http');
+const http = require('http');
+const { convertToCase } = require('../src/convertToCase');
 
-function request(url = '/') {
-  return new Promise((resolve, reject) => {
-    get('http://localhost:5701' + url, (res) => {
-      let body = '';
+const createServer = () => {
+  const availableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
 
-      res.on('data', (chunk) => {
-        body += chunk;
-      });
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
 
-      res.on('end', () => {
-        resolve({ res, body });
-      });
-    })
-      .on('error', (err) => {
-        reject(err);
-      });
-  });
-}
+    const normalizedURL = new URL(
+      req.url, `http://localhost${req.headers.host}`,
```

Comment:
The URL constructor is not properly initiated. The base URL should be `http://${req.headers.host}`